src/profstats/main.c

```diff
@@ -344,7 +344,7 @@ int main(int argc, char **argv) {
 		if (cnt_seccomp > (seccomp + 1))
 			cnt_seccomp = seccomp + 1;
 		if (cnt_restrict_namespaces > (restrict_namespaces + 1))
-			cnt_seccomp = restrict_namespaces + 1;
+			cnt_restrict_namespaces = restrict_namespaces + 1;
```

Comment:
This looks good but it's unrelated to everything else; please do it in a
separate PR.

Reply:
Okay, see #6369